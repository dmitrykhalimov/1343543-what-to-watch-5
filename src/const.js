export const ratingRanks = [
  {
    'Bad': 2.9
  },
  {
    'Normal': 4.9
  },
  {
    'Good': 7.9
  },
  {
    'Very good': 9.9
  },
  {
    'Awesome': 10
  }
];
