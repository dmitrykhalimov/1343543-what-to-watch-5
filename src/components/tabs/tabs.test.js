import React from "react";
import renderer from "react-test-renderer";
import Tabs from "./tabs";
import {TEST_MOCKS} from "../../const";

const mockComponent = () => {
  return (
    <div>
    </div>
  );
};

describe(`Tabs`, () => {
  it(`Should Tabs render correctly`, () => {
    const tree = renderer
      .create(
          <Tabs
            activeTab={TEST_MOCKS.tab.OVERVIEW}
            onSwitchTab={TEST_MOCKS.noop}
            poster={TEST_MOCKS.poster}
            tabToRender={mockComponent()}
            tabs={Array.from(Object.values(TEST_MOCKS.tab.OVERVIEW))}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
