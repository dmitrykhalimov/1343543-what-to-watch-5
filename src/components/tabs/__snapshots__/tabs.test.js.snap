// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs Should Tabs render correctly 1`] = `
<div
  className="movie-card__info"
>
  <div
    className="movie-card__poster movie-card__poster--big"
  >
    <img
      alt="The Grand Budapest Hotel poster"
      height="327"
      src="https://m.media-amazon.com/images/M/MV5BZDYwYzlhOTAtNDAwMC00ZTBhLWI4M2QtMTA1NmJhYTdiNTkxXkEyXkFqcGdeQXVyNTM0NTU5Mg@@._V1_UX182_CR0,0,182,268_AL_.jpg"
      width="218"
    />
  </div>
  <div
    className="movie-card__desc"
  >
    <nav
      className="movie-nav movie-card__nav"
    >
      <ul
        className="movie-nav__list"
      >
        <li
          className="movie-nav__item"
          onClick={[Function]}
        >
          <a
            className="movie-nav__link"
            href="#"
          >
            O
          </a>
        </li>
        <li
          className="movie-nav__item"
          onClick={[Function]}
        >
          <a
            className="movie-nav__link"
            href="#"
          >
            V
          </a>
        </li>
        <li
          className="movie-nav__item"
          onClick={[Function]}
        >
          <a
            className="movie-nav__link"
            href="#"
          >
            E
          </a>
        </li>
        <li
          className="movie-nav__item"
          onClick={[Function]}
        >
          <a
            className="movie-nav__link"
            href="#"
          >
            R
          </a>
        </li>
        <li
          className="movie-nav__item"
          onClick={[Function]}
        >
          <a
            className="movie-nav__link"
            href="#"
          >
            V
          </a>
        </li>
        <li
          className="movie-nav__item"
          onClick={[Function]}
        >
          <a
            className="movie-nav__link"
            href="#"
          >
            I
          </a>
        </li>
        <li
          className="movie-nav__item"
          onClick={[Function]}
        >
          <a
            className="movie-nav__link"
            href="#"
          >
            E
          </a>
        </li>
        <li
          className="movie-nav__item"
          onClick={[Function]}
        >
          <a
            className="movie-nav__link"
            href="#"
          >
            W
          </a>
        </li>
      </ul>
    </nav>
    <div />
  </div>
</div>
`;
